packages/cli/src/util/login/email.ts

```diff
@@ -51,9 +53,9 @@ export default async function doEmailLogin(
         'Email',
         ssoUserId
       );
-    } catch (err) {
-      if (err.serverMessage !== 'Confirmation incomplete') {
-        output.error(err.message);
+    } catch (err: unknown) {
+      if (!isAPIError(err) || err.serverMessage !== 'Confirmation incomplete') {
```

Comment:
This seems wrong since the API error would not print an error and instead print success

Reply:
It's using `||` here, so it _will_ print the error message so long as `serverMessage !== 'Confirmation incomplete'`.